src/app-layout/runtime-api.tsx

```diff
@@ -28,8 +28,17 @@ export interface DrawersLayout {
 }
 
 export function convertRuntimeDrawers(drawers: Array<RuntimeDrawerConfig>): DrawersLayout {
-  const converted = drawers.map(({ mountContent, unmountContent, ...runtimeDrawer }) => ({
+  const converted = drawers.map(({ mountContent, unmountContent, trigger, ...runtimeDrawer }) => ({
     ...runtimeDrawer,
+    trigger: trigger.iconSvg
+      ? {
+          ...trigger,
+          iconSvg: (
+            // eslint-disable-next-line react/no-danger
+            <svg viewBox="0 0 16 16" focusable="false" dangerouslySetInnerHTML={{ __html: trigger.iconSvg }}></svg>
```

Comment:
I explored this option, it does not work as you could expect. Because if not `<svg>`, I will have to add a wrapper `<span>` node which won't support this feature

https://github.com/cloudscape-design/components/blob/f9029ce3e6cc9c6b859b2cbbece2c21c3bd1e4d1/src/icon/interfaces.ts#L48-L54

I think, this is an acceptable trade-off, I will mention this in the `registerDrawer` docs, that only "bring your own complete SVG" mode works

Reply:
Ah okay, I see what you're saying. I guess that's fine then.